GVFS/GVFS.FunctionalTests/Tests/GitCommands/CheckoutTests.cs

```diff
@@ -498,6 +500,12 @@ public void CheckoutBranchWithOpenHandleBlockingRepoMetdataUpdate()
         [TestCase]
         public void CheckoutBranchWithOpenHandleBlockingProjectionDeleteAndRepoMetdataUpdate()
         {
+            bool unregisterOfflineIO = false;
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.OSX))
+            {
+                unregisterOfflineIO = 0 == RegisterForOfflineIO();

```

Comment:
To be more in line with our C# syntax I would define 0 at the top of the file
```
private const int success = 0;
```

Then just fail if we can't register.  It will make it easier to debug the functional test that way 
```
If RegisterForOfflineIO() != success)
{
   Assert.Fail("Could not register for OfflineIO();
}
```

Reply:
Yes, that makes more sense. I've updated the code to something along those lines.